samples/command/provider/src/provider_impl.rs

```diff
@@ -89,7 +91,7 @@ impl Provider for ProviderImpl {
         info!("Notification: '{payload}'");
 
         tokio::spawn(async move {
-            let client_result = ConsumerClient::connect(consumer_uri.clone()).await;
+            let client_result = DigitalTwinConsumerClient::connect(consumer_uri.clone()).await;
```

Comment:
You can replace this, the if statement, and the client_result.unwrap() with this ``DigitalTwinConsumerClient::connect(consumer_uri.clone()).await.map_err(|error| Status::internal(format!("{error:?}")))?;``

Reply:
Fixed.